functions/src/Controller/Committee.ts

```diff
@@ -51,4 +51,20 @@ export const deleteCommitteeController = functions.https.onRequest(async (reques
 			response.status(404).send('Error 404: Committee document not found');
 		}
 	});
+});
+
+export const changeLevelController = functions.https.onRequest(async (request, response) => {
+	const committee: Committee = new Committee(request.body);
+	const committeeCollection = getCommitteeCollection();
+	const committeeDoc = committeeCollection.doc(committee.id);
+
+	await committeeDoc.get().then((docSnapshot) => {
+		if (docSnapshot.exists) {
+			void changeLevel(committee);
```

Comment:
you don't need the void here as well I think it is a bit unnecessary because if we add return type to your function we will have to add them to all functions to maintain style.